rpcserver.go

```diff
@@ -39,6 +40,28 @@ import (
 
 var (
 	defaultAccount uint32 = waddrmgr.DefaultAccountNum
+
+	// roMethods is a slice of method names that are considered "read-only"
+	// for authorization purposes, all lowercase.
+	roMethods = []string{
```

Comment:
The following methods I consider to be read-only are missing from this list: 
 * `SubscribeTransactions`
 * `SubscribeInvoices` 
 * `SubscribeChannelGraph`